src/services/GraphInit.jl

```diff
@@ -239,6 +239,20 @@ function initVariable!( dfg::AbstractDFG,
   return nothing
 end
 
+function initVariable!( dfg::AbstractDFG, 
```

Comment:
id also do both the (::DFGVariable, args...) and
(grtVariable(dfg,lb), args...)

dispatches